service/controller/v16/resource/cloudformation/current.go

```diff
@@ -99,6 +100,32 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		} else if err != nil {
 			return StackState{}, microerror.Mask(err)
 		}
+
+		var workerDockerVolumeSizeGB int
+		{
+			v, err := sc.CloudFormation.GetOutputValue(stackOutputs, key.WorkerDockerVolumeSizeKey)
+			if cloudformationservice.IsOutputNotFound(err) {
+				// Since we are transitioning between versions we will have situations in
+				// which old clusters are updated to new versions and miss the docker
+				// volume resource name in the CF stack outputs. We ignore this problem
+				// for now and move on regardless. On the next resync period the output
+				// value will be there, once the cluster got updated.
+				//
+				// TODO remove this condition as soon as all guest clusters in existence
+				// obtain a docker volume size.
+				v = "100"
```

Comment:
Lets please have a TODO issue for this.

Reply:
https://github.com/giantswarm/giantswarm/issues/4139